src/app/containers/VestedHistory/components/VestedHistoryTable.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { VestedHistoryTableRow } from './VestedHistoryTableRow';
+
+interface IVestedHistoryTable {
+  items: any;
+}
+
+export const VestedHistoryTable: React.FC<IVestedHistoryTable> = ({
+  items,
+}) => {
+  return (
+    <>
+      {items.map((item, index) => {
+        return (
+          <VestedHistoryTableRow
+            key={item.transactionHash + index}
+            item={item}
+          />
+        );
+      })}
+    </>
+  );
+};
```

Comment:
```suggestion
export const VestedHistoryTable: React.FC<IVestedHistoryTable> = ({
  items,
}) => (
  <>
    {items.map((item, index) => (
      <VestedHistoryTableRow key={item.transactionHash + index} item={item} />
    ))}
  </>
);
```

Reply:
Btw do you need to have a composite key (`item.transactionHash + index`) ?

Reply:
I think the index will be enough